packages/manager/cypress/e2e/core/linodes/create-linode.spec.ts

```diff
@@ -105,4 +129,87 @@ describe('create linode', () => {
           .click();
       });
   });
+
+  /*
+   * - Confirms DC-specific pricing UI flow works as expected during Linode creation.
+   * - Confirms that pricing notice is shown in "Region" section.
+   * - Confirms that notice is shown when selecting a region with a different price structure.
+   */
+  it('shows DC-specific pricing information during create flow', () => {
+    const rootpass = randomString(32);
+    const linodeLabel = randomLabel();
+    const initialRegion = getRegionById('us-west');
+    const newRegion = getRegionById('us-east');
+
+    const mockLinode = linodeFactory.build({
+      label: linodeLabel,
+      region: initialRegion.id,
+      type: dcPricingMockLinodeTypes[0].id,
+    });
+
+    mockAppendFeatureFlags({
+      dcSpecificPricing: makeFeatureFlagData(true),
+    }).as('getFeatureFlags');
+    mockGetFeatureFlagClientstream().as('getClientStream');
+
+    // Mock requests to get individual types.
+    mockGetLinodeType(dcPricingMockLinodeTypes[0]);
+    mockGetLinodeType(dcPricingMockLinodeTypes[1]);
+    mockGetLinodeTypes(dcPricingMockLinodeTypes).as('getLinodeTypes');
+
+    // intercept request
+    cy.visitWithLogin('/linodes/create');
+    cy.wait(['@getClientStream', '@getFeatureFlags', '@getLinodeTypes']);
+
+    mockCreateLinode(mockLinode).as('linodeCreated');
+    cy.get('[data-qa-header="Create"]').should('have.text', 'Create');
+    getClick('[data-qa-deploy-linode]');
+
+    // A message is shown to instruct users to select a region in order to view plans and prices
+    cy.get('[data-qa-tp="Linode Plan"]').should(
+      'contain.text',
+      'Plan is required.'
+    );
+    cy.get('[data-qa-tp="Linode Plan"]').should(
+      'contain.text',
+      dcPricingPlanPlaceholder
+    );
+
+    // Confirm that the checkout summary at the bottom of the page reflects the correct price.
+    containsClick(selectRegionString).type(`${initialRegion.label} {enter}`);
+    fbtClick('Shared CPU');
+    getClick('[id="g6-standard-8"]');
```

Comment:
```suggestion
    getClick(`[id="${dcPricingMockLinodeTypes[0].id}"]`);
```
Coincidentally, I just posted [an explanation on another PR](https://github.com/linode/manager/pull/9707#discussion_r1333483107) about this and why hardcoding the ID here might make the tests fragile in unexpected ways!